src/CodeGen_Hexagon.cpp

```diff
@@ -1900,6 +1906,35 @@ void CodeGen_Hexagon::visit(const Call *op) {
         return;
     }
 
+    if (op->is_intrinsic("gather")) {
```

Comment:
is_intrinsic compares by pointer, so it's not really safe to use string literals. It's designed for the static members of Call of type ConstString